mainnet-dryrun-2/gentx/gentx-36e26d2bbb90b12a0246fe832a3b4076df1b2442.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"DSRV","identity":"CC434B6FE536F51B","website":"https://dsrvlabs.com","security_contact":"validator@dsrvlabs.com","details":"DSRV is an integrated blockchain solutions company with the mission of enriching the crypto ecosystem via stronger connectivity. We strive to be your gateway to a suite of all-comprehensive blockchain services. Everything distributed, served complete."},"commission":{"rate":"0.010000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.100000000000000000"},"min_self_delegation":"1","delegator_address":"archway1pm0yyd2ncc2x67ctuz5p3tcxa59tezx5scp0hj","validator_address":"archwayvaloper1pm0yyd2ncc2x67ctuz5p3tcxa59tezx5sugd6m","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"FCDoXNkWW6h5yB5h9F46R7gVVdgvqB2adObXkjhLCv4="},"value":{"denom":"aarch","amount":"9500000000000000000"}}],"memo":"36e26d2bbb90b12a0246fe832a3b4076df1b2442@137.184.234.151:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AuWQUOm7PfEQgoszl3lryD3gE86exQKrqcwzm9G0CVdC"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[{"denom":"aarch","amount":"180000000000000000"}],"gas_limit":"200000","payer":"","granter":""}},"signatures":["9cNIWmdsy90b6AOfq2+WWp9P8AycAnWdJelvk+I/zi9QLEsC88Ihg7q0BQOVUnbV4nepmVDFMV4O56weouIuQA=="]}
```

Comment:
Validation fails with
```
panic: signature verification failed; please verify account number (0) and chain-id (mainnet-dry-run-2): unauthorized
```
Was genesis.json replaced with unpacked https://github.com/archway-network/mainnet-dryrun/blob/main/mainnet-dryrun-2/init_genesis.json.gz before doing gentx?

Reply:
I'll check again push this ASAP.

Reply:
@kayano I make mistake by using chain ID as `mainnet-dryrun-2`. Now chain ID fixed to `mainnet-dry-run-2` at e1e2268dbaf516ee3294b65b7ac087d22dddb478